airbyte-integrations/connectors/source-stripe/setup.py

```diff
@@ -30,5 +30,5 @@
     author_email="contact@airbyte.io",
     packages=find_packages(),
     package_data={"": ["*.json", "schemas/*.json", "schemas/shared/*.json"]},
-    install_requires=["airbyte-protocol", "base-python", "stripe"],
+    install_requires=["airbyte-cdk==0.1.0rc5", "stripe"],
```

Comment:
this will be changed after all PRs are merged and the non-rc version is published